publication-commons/src/main/java/no/unit/nva/publication/model/business/importcandidate/NotApplicable.java

```diff
@@ -0,0 +1,74 @@
+package no.unit.nva.publication.model.business.importcandidate;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonTypeInfo;
+import no.unit.nva.model.Username;
+import nva.commons.core.JacocoGenerated;
+
+import java.time.Instant;
+import java.util.Objects;
+
+@JsonTypeInfo(use = JsonTypeInfo.Id.NAME, property = "type")
+public class NotApplicable implements ImportStatus {
+
+    private final static String SET_TO_NOT_APPLICABLE_BY = "setToNotApplicableBy";
+    private final static String COMMENT = "comment";
+    private final static String SET_TO_NOT_APPLICABLE_DATE = "setToNotApplicableDate";
+
+    @JsonProperty(SET_TO_NOT_APPLICABLE_BY)
+    private final Username setToNotApplicableBy;
+
+    @JsonProperty(COMMENT)
+    private final String comment;
+
+    @JsonProperty(SET_TO_NOT_APPLICABLE_DATE)
+    private final Instant setToNotApplicableDate;
```

Comment:
Disse variabel navnene synes jeg selv er skrot, men kommer ikke på noe bedre. Har dere bedre forslag?

Reply:
Litt lange navn enig, kanskje setDate og setBy, siden de skal ligge inne ImportStatus objektet, så gir de mening:
```

ImportStatus {
"setBy": "asdad",
"setDate": "20.20.2022",
"comment": "sdlmøbporkhpoekpoOPFn wpejgpwgo"
}
```

Reply:
createdBy / modifiedBy
created / modified

Reply:
Also, there are simpler ways to implement serializable dto's
```
public class RetentionStrategyDto implements JsonSerializable {

    private final RetentionStrategyType retentionStrategy;
    private final URI id;

    @ConstructorProperties({"retentionStrategy", "id"})
    public RetentionStrategyDto(RetentionStrategyType retentionStrategy, URI id) {
        this.retentionStrategy = retentionStrategy;
        this.id = id;
    }

```    
add getters and equal+hash+tostring to complete.

Reply:
less is more

Reply:
Trying out setDate, setBy. Hope it doesn't destroy the JSON-ld context like "other" does.